CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ if(HPCC_USE_X86_64)
 endif()
 
 if(HPCC_USE_CUDA)
+    if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.18.0")
+        cmake_policy(SET CMP0104 OLD)
```

Comment:
Can this rule apply to other platforms(x86, arm, etc)? If so, this rule should be placed outside the `if(HPCC_USE_CUDA)` clause.